spec/forms/teacher_interface/alternative_name_form_spec.rb

```diff
@@ -7,10 +7,8 @@
 
   let(:application_form) { build(:application_form) }
 
-  describe "#valid?" do
-    subject(:valid?) { form.valid? }
-
-    let(:form) do
+  describe "#validation?" do
```

Comment:
```suggestion
  describe "validations" do
```

Reply:
You can also merge this section with the describe block at the top of this file.